src/components/draw/feature-edit-style.vue

```diff
@@ -52,31 +95,52 @@ function onClickChangeLineStyle(style: string) {
           <input
             class="cursor-pointer"
             type="color"
-            value=""
+            :value="feature.featureStyle.color"
+            @input="onColorSelect"
             data-cy="featStyleColor"
           />
         </div>
       </div>
     </template>
 
-    <template v-slot:size>
+    <template v-slot:size="slotProps">
       <div class="flex gap-1 items-center mt-1">
         <label class="font-bold block" for="inline-full-name">
           {{ t('Size') }}
         </label>
-        <RangeInput class="md:w-2/3" :max="900" data-cy="featStyleSize" />
+        <RangeInput
+          class="md:w-2/3"
+          :max="slotProps.maxsize"
+          :value="feature.featureStyle.size"
+          data-cy="featStyleSize"
+          @change="onSizeChange"
+        />
       </div>
     </template>
 
     <template v-slot:angle>
-      <div class="flex gap-1 items-center mt-1 mb-2">
+      <div
+        v-if="
+          !(
+            feature.featureType == 'drawnPoint' &&
```

Comment:
Use `===`

Reply:
OK

Reply:
Still "==" in your code

Reply:
I grepped on .ts files, not on .vue, zut!